tutorials/run-hapijs-on-google-app-engine.md

```diff
@@ -1,17 +1,14 @@
 ---
-title: Run Hapi.js on Google App Engine Flexible Environment
```

Comment:
According to branding guidelines it should be Run Hapi.js on the App Engine flexible environment